eclair-core/src/main/scala/fr/acinq/eclair/db/RelayFeesDb.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.db
+
+import fr.acinq.bitcoin.Crypto.PublicKey
+import fr.acinq.eclair.MilliSatoshi
+
+import java.io.Closeable
+
+trait RelayFeesDb extends Closeable {
```

Comment:
What is the rationale for not adding this to the `AuditDb`? It would fit right in I think.

Reply:
`AuditDb` should contain information for auditing only and should not be required for running the node, it should be write only. This new database is required for running the node as eclair reads from it.

Reply:
> eclair reads from it

I missed that! In that case, it belongs to the `PeersDb` I think. There is an ambiguity though, because we are also using that same table as an append-only, timestamped, event log. How about a very basic key-value table in `PeersDb` and an event log table in `AuditDb`? Yes there is duplication, but it's consistent with what we have between `PaymentsDb` and `AuditDb`.

We could also just put the relay fee changes in the `channel_event` table, or introduce a new `peer_event` table.

Reply:
Done, I've added a `channel_updates` table in `AuditDb`.